### PR TITLE
WIP: Set the AnP environment before restarting a build.

### PR DESCRIPTION
**Pull request by jasonwalker80**
Tests fail since the AnP used doesn't have config. Is there an example of configuring a test AnP to have the environment config?

### CODE REVIEW

**Comment by apipe-tester** [comment]
Merged build finished. Test PASSed.

**Comment by apipe-tester** [comment]
Merged build finished. Test FAILed.

File: lib/perl/Genome/Model/CwlPipeline/Command/Restart.pm [hunk 1]
```diff
@@ -32,6 +32,16 @@ sub execute {
     my $build_count = scalar(@builds);
     my @errors;
     for my $build (@builds) {
+        my $anp = $build->model->analysis_project;
```
Comment (tmooney) [inline lib/perl/Genome/Model/CwlPipeline/Command/Restart.pm]: Should we put this inside `_restart_build` so that each build can be independently tried?

**Comment by jasonwalker80** [comment]
Closing in favor of #1828

**Pull request closed without merging**